src/android/app/src/main/java/io/github/lime3ds/android/fragments/EmulationFragment.kt

```diff
@@ -769,55 +784,96 @@ class EmulationFragment : Fragment(), SurfaceHolder.Callback, Choreographer.Fram
         popupMenu.show()
     }
 
-    private fun showScreenLayoutMenu() {
+    private fun showLandscapeScreenLayoutMenu() {
         val popupMenu = PopupMenu(
             requireContext(),
             binding.inGameMenu.findViewById(R.id.menu_landscape_screen_layout)
         )
 
         popupMenu.menuInflater.inflate(R.menu.menu_landscape_screen_layout, popupMenu.menu)
 
-        val layoutOptionMenuItem = when (EmulationMenuSettings.landscapeScreenLayout) {
+        val layoutOptionMenuItem = when (IntSetting.SCREEN_LAYOUT.int) {
             ScreenLayout.SINGLE_SCREEN.int ->
                 R.id.menu_screen_layout_single
 
             ScreenLayout.SIDE_SCREEN.int ->
                 R.id.menu_screen_layout_sidebyside
 
-            ScreenLayout.MOBILE_PORTRAIT.int ->
-                R.id.menu_screen_layout_portrait
-
             ScreenLayout.HYBRID_SCREEN.int ->
                 R.id.menu_screen_layout_hybrid
 
+            ScreenLayout.CUSTOM_LAYOUT.int ->
+                R.id.menu_screen_layout_custom
+
             else -> R.id.menu_screen_layout_landscape
         }
         popupMenu.menu.findItem(layoutOptionMenuItem).setChecked(true)
 
         popupMenu.setOnMenuItemClickListener {
             when (it.itemId) {
                 R.id.menu_screen_layout_landscape -> {
-                    screenAdjustmentUtil.changeScreenOrientation(ScreenLayout.MOBILE_LANDSCAPE)
-                    true
-                }
-
-                R.id.menu_screen_layout_portrait -> {
-                    screenAdjustmentUtil.changeScreenOrientation(ScreenLayout.MOBILE_PORTRAIT)
+                    screenAdjustmentUtil.changeScreenOrientation(ScreenLayout.MOBILE_LANDSCAPE.int)
                     true
                 }
```

Comment:
There is not longer a MOBILE_PORTRAIT option in the standard ScreenLayout enum, because there is not an additional PortraitScreenLayout enum that is used instead when the screen is in portrait mode. So the former ScreenLayout.MOBILE_PORTRAIT has been replaced with PortraitScreenLayout.TopFullWidth

Reply:
Ok, but that's not what this was for
This removed enum was a landscape layout which mimicked the default portrait layout. It now no longer exists. This is what it looked like:
![Screenshot_20240728-142615_Lime3DS.jpg](https://github.com/user-attachments/assets/d189329e-3db4-46b2-b897-83c8d7db6823)

Reply:
How is that significantly different than the "default" vertical layout that is currently (for some reason) ignored on android? That one is still available, though not currently exposed in the UI. I think it would make more sense to expose that one on android than keep a layout labeled "Portrait" that can only be used in Landscape.

Reply:
I can add the MobilePortrait back in if you think its best, but for parity across desktop and android I think a better choice to get a vertical layout is to re-expose the desktop Default layout on Android (and re-label it Vertical or Top/Bottom or something similar).

Similarly (but a different PR, I think) I think it would be best to remove the MobileLandscape layout and instead simply use the desktop Large Screen for that purpose - the code could be written to detect Android and push the small screen to the top instead of bottom, which is the only significant difference between the two layouts other than proportional size, which can be handled by setting a different default value for the "Large Screen Proportion" setting on android.

Reply:
Both of these suggestions sound fine to me, I just didn't want this layout to be outright unaccessible under whatever form you chose as it was in the original PR

Reply:
Added the vetical layout back in. Two screenshots below show the updated menu and selected layout.
![WhatsApp Image 2024-07-28 at 16 21 24 (1)](https://github.com/user-attachments/assets/6738e43b-8d22-4a9a-96d6-946273761ce0)

![WhatsApp Image 2024-07-28 at 16 21 24](https://github.com/user-attachments/assets/5769769b-1598-4272-9540-dbe154fc7bfa)